nextjs/src/app/(app)/data-sources/create/connect/[externalDataSourceType]/page.tsx

```diff
@@ -94,23 +95,7 @@ export default function Page({
 
   const form = useForm<FormInputs>({
     defaultValues: {
-      name: '',
       geographyColumnType: GeographyTypes.Postcode,
```

Comment:
suggestion: we should keep these default variables to squash console error logs, but restrict them based on the externalDataSourceType.

Reply:
OK, I can revert these changes.

Reply:
Fixed. Noticed a separate bug where postcode field doesn't get prefilled anymore by guesser.